indico/modules/events/layout/templates/css_preview_header.html

```diff
@@ -0,0 +1,23 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{{ form_header(form, action=url_for('event_layout.css_save_theme', event)) }}
+{{ form_rows(form) }}
+{% call form_footer(form) %}
+    <input class="i-button highlight" type="submit" value="{% trans %}Use selected template{% endtrans %}">
+    <a href="{{ url_for('event_layout.index', event) }}" class="i-button icon-arrow-left" data-button-back>
+        {%- trans %}Back to management area{% endtrans -%}
+    </a>
+    {% if download_url %}
+        <a href="{{ download_url }}" class="i-button icon-file-download">{% trans %}Download CSS{% endtrans %}</a>
+    {% endif %}
+{% endcall %}
+<script>
+    $(document).ready(function() {
+        $('#theme').nullableselector({nullvalue: ''})
```

Comment:
every time you forget a semicolon, someone sacrifices a little kitten to a catholic priest :cat: